### ISSUE TITLE
✅ 🐞 Bug report: Video not opening in lightbox

### ISSUE THREAD

**Issue by NickNZ01** [elementor/elementor#18211]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Video not opening in lightbox. Tried in various places, as action on a button, action on a hotspot etc. Video not showing. Error message: The embed code for this video is not valid

### Steps to reproduce

Screencast: https://share.getcloudapp.com/mXuYqQng

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently using the steps above.

### System Info

<details>
<summary>Click to reveal</summary>

````txt


					
== Server Environment ==
	Operating System: Linux
	Software: Apache
	MySQL version: MariaDB Server v10.3.34
	PHP Version: 7.4.28
	PHP Max Input Vars: 1000
	PHP Max Post Size: 8M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 5.9.2
	Site URL: https://campervan-hire-new-zealand.com
	Home URL: https://campervan-hire-new-zealand.com
	WP Multisite: No
	Max Upload Size: 2 MB
	Memory limit: 256M
	Permalink Structure: /%postname%/
	Language: en-NZ
	Timezone: Pacific/Auckland
	Debug Mode: Inactive

== Theme ==
	Name: BOICampervans
	Version: 1.0.0
	Author: Nick Galama
	Child Theme: Yes
	Parent Theme Name: Astra
	Parent Theme Version: 3.7.9
	Parent Theme Author: Brainstorm Force

== User ==
	Role: administrator
	WP Profile lang: en_NZ
	User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/99.0.4844.74 Safari/537.36

== Active Plugins ==
	Advanced Custom Fields
		Version: 5.12
		Author: Delicious Brains

	Astra Pro
		Version: 3.6.6
		Author: Brainstorm Force

	Broken Link Checker
		Version: 1.11.16
		Author: WPMU DEV

	Cookie Notice & Compliance for GDPR / CCPA
		Version: 2.2.3
		Author: Hu-manity.co

	Custom Post Type UI
		Version: 1.11.1
		Author: WebDevStudios

	DVFAQ
		Version: 1.2
		Author: Egemenerd

	Elementor
		Version: 3.5.6
		Author: Elementor.com

	Elementor Pro
		Version: 3.6.4
		Author: Elementor.com

	JetSearch For Elementor
		Version: 2.1.15
		Author: Crocoblock

	Premium Addons for Elementor
		Version: 4.9.2
		Author: Leap13

	Premium Addons PRO
		Version: 2.7.4
		Author: Leap13

	Rank Math SEO
		Version: 1.0.84
		Author: Rank Math

	Rank Math SEO PRO
		Version: 3.0.8
		Author: Rank Math

	ShortPixel Image Optimizer
		Version: 4.22.8
		Author: ShortPixel

	Turn Rank Math FAQ Block to Accordion
		Version: 1.1.0
		Author: WPHowKnow

	Wordfence Security
		Version: 7.5.8
		Author: Wordfence

	WP Rocket
		Version: 3.10.9
		Author: WP Media

	WP Rollback
		Version: 1.7.1
		Author: Impress.org


== Elements Usage ==
	
	footer : 1
		icon-list : 4
		social-icons : 1
	header : 2
		button : 2
		nav-menu : 2
		theme-site-logo : 2
	page : 9
		accordion : 1
		button : 2
		divider : 2
		form : 2
		heading : 17
		image : 9
		menu-anchor : 3
		premium-addon-image-hotspots : 4
		premium-google-reviews : 2
		premium-image-accordion : 1
		premium-unfold-addon : 2
		slides : 2
		spacer : 5
		text-editor : 30
	popup : 2
		html : 2
	section : 18
		accordion : 1
		button : 2
		heading : 3
		html : 2
		image : 9
		posts : 1
		premium-google-reviews : 1
		premium-image-accordion : 2
		spacer : 5
		text-editor : 26
		video : 2
	single : 2
		divider : 1
		heading : 1
		jet-ajax-search : 1
		posts : 2
		share-buttons : 1
		spacer : 3
		theme-post-content : 2
		theme-post-excerpt : 1
		theme-post-featured-image : 1
		theme-post-title : 2
	widget : 0
		form : 2
		premium-google-reviews : 1
	wp-post : 11
		heading : 1
		image : 1
		posts : 1
		spacer : 1
		text-editor : 13
		video : 2
	wp-page : 21
		accordion : 1
		button : 4
		divider : 4
		dvfaq-cat : 1
		form : 2
		google_maps : 1
		heading : 66
		image : 20
		jet-ajax-search : 1
		menu-anchor : 6
		posts : 2
		premium-addon-image-hotspots : 2
		premium-addon-tabs : 2
		premium-facebook-reviews : 1
		premium-google-reviews : 2
		premium-tables-addon : 1
		premium-unfold-addon : 1
		shortcode : 1
		slides : 1
		spacer : 44
		text-editor : 74
		theme-post-title : 1
		video : 1
	single-post : 2
		heading : 2
		posts : 1
		share-buttons : 1
		theme-post-content : 1
		theme-post-featured-image : 1
		theme-post-title : 1


== Elementor Experiments ==
	Optimized DOM Output: Active
	Improved Asset Loading: Active
	Improved CSS Loading: Inactive by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active
	Import Export Template Kit: Active by default
	Additional Custom Breakpoints: Inactive by default
	Hide native WordPress widgets from search results: Active by default
	Default to New Theme Builder: Active by default
	Landing Pages: Active by default
	Color Sampler: Active by default
	Favorite Widgets: Active by default
	Admin Top Bar: Active by default
	Page Transitions: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default


== Log ==
	
Log: showing 20 of 382022-01-16 09:29:01 [info] Elementor/Upgrades - _v_3_5_0_remove_old_elementor_scheme Start  
2022-01-16 09:29:01 [info] Elementor/Upgrades - _v_3_5_0_remove_old_elementor_scheme Finished 
2022-01-16 09:29:01 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.4.8',
  'to' => '3.5.3',
)]
2022-01-16 09:29:01 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.4.8',
  'to' => '3.5.3',
)]
2022-03-12 14:23:11 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.5.3',
  'to' => '3.5.6',
)]
2022-03-12 14:23:11 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.5.2',
  'to' => '3.6.3',
)]
2022-03-12 14:23:12 [info] elementor::elementor_updater Started 
2022-03-12 14:23:12 [info] Elementor/Upgrades - _on_each_version Start  
2022-03-12 14:23:12 [info] Elementor/Upgrades - _on_each_version Finished 
2022-03-12 14:23:12 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor',
  'from' => '3.5.3',
  'to' => '3.5.6',
)]
2022-03-12 14:23:14 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor',
  'from' => '3.5.3',
  'to' => '3.5.6',
)]
2022-03-12 14:23:19 [info] elementor-pro::elementor_pro_updater Started 
2022-03-12 14:23:19 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-03-12 14:23:19 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-03-12 14:23:19 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.5.2',
  'to' => '3.6.3',
)]
2022-03-18 15:05:22 [info] elementor-pro::elementor_pro_updater Started 
2022-03-18 15:05:22 [info] Elementor Pro/Upgrades - _on_each_version Start  
2022-03-18 15:05:22 [info] Elementor Pro/Upgrades - _on_each_version Finished 
2022-03-18 15:05:22 [info] Elementor data updater process has been completed. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.6.3',
  'to' => '3.6.4',
)]
2022-03-18 15:05:23 [info] Elementor data updater process has been queued. [array (
  'plugin' => 'Elementor Pro',
  'from' => '3.6.3',
  'to' => '3.6.4',
)]

PHP: showing 4 of 4PHP: 2021-12-04 14:09:16 [notice X 96][../wp-content/plugins/elementor/includes/managers/image.php::34] Undefined index: size [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-01-16 14:09:53 [notice X 440][../wp-content/plugins/elementor-pro/modules/forms/classes/form-record.php::179] Undefined index: referrer [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-03-05 12:40:17 [notice X 5][../wp-content/plugins/elementor/core/common/modules/ajax/module.php::171] Undefined index: data [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]
PHP: 2022-03-05 19:12:27 [notice X 55][../wp-content/plugins/elementor-pro/modules/dynamic-tags/tags/post-featured-image.php::36] Trying to access array offset on value of type bool [array (
  'trace' => '
#0: Elementor\Core\Logger\Manager -> shutdown()
',
)]



== Elementor - Compatibility Tag ==
	
	Elementor Pro: Compatible
	JetSearch For Elementor: Incompatible
	Premium Addons for Elementor: Compatible
	Premium Addons PRO: Compatible

== Elementor Pro - Compatibility Tag ==
	
	JetSearch For Elementor: Incompatible
	Premium Addons for Elementor: Compatible
	Premium Addons PRO: Compatible
				

````

</details>

**Comment by steveni1963** [comment]
Bug #17619 seems to be the same issue.  The quick solution posted in #17619 still works.

**Comment by nicholaszein** [comment]
Hello!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor v3.23.0! 🥳 

Now Vimeo videos that have privacy settings enabled should load correctly on the lightbox.

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
